src/AudioFileSourceHTTPStream.cpp

```diff
@@ -24,16 +24,58 @@
 
 AudioFileSourceHTTPStream::AudioFileSourceHTTPStream()
 {
+ // init_ascii_to_hex();
   pos = 0;
   reconnectTries = 0;
   saveURL[0] = 0;
+  next_chunk = 0;
 }
 
 AudioFileSourceHTTPStream::AudioFileSourceHTTPStream(const char *url)
 {
+  //init_ascii_to_hex();
   saveURL[0] = 0;
   reconnectTries = 0;
+  next_chunk = 0;
   open(url);
+
+}
+
+int AudioFileSourceHTTPStream::convertHexToInt(const char *str)
```

Comment:
Why not just `int(maybe unsigned int?) val; auto ret = sscanf(str, "%x", &val);  if (ret) return ret; else { ..log error return -1...}`?  That would drop the std::map need

Reply:
Fixed !